docs/quick-start/2-wallet-setup.md

```diff
@@ -20,37 +20,30 @@ TBD
 
 - Installation: MetaMask can be installed from their [official website](https://metamask.io/download/).
 - Importing Configurations: To set up MetaMask for Morph Testnet, click the "add to wallet" buttons on the Morph Testnet homepage. This will automatically import the chain ID and RPC URLs for the Morph Testnet.
-- Using Ethereum Sepolia Testnet: Morph Testnet utilizes the Ethereum Sepolia testnet as its underlying L1, which is already configured in MetaMask by default. To access it, enable "Show/hide test networks" in the MetaMask network selection dropdown.
+- Using Ethereum Holesky Testnet: Morph Testnet utilizes the Ethereum Holesky testnet as its underlying L1, which is already configured in MetaMask by default. To access it, enable "Show/hide test networks" in the MetaMask network selection dropdown.
 
 
 
 ### Manual network configuration
 
-Currently, the Add to wallet links may not be compatible with all wallets yet. If you are having issues using them, you may need to manually add the Sepolia Testnet and Morph by inserting the configuration details from the table below:
+Currently, the Add to wallet links may not be compatible with all wallets yet. If you are having issues using them, you may need to manually add the Holesky Testnet and Morph by inserting the configuration details from the table below:
 
 
 #### Network Configuration
 
+:::tip Morph Sepolia Sunset
 
-:::tip
-
-You can use any of the following 3 rpc for connections:
-
-~~~
-https://rpc-testnet.morphl2.io
-
-https://rpc1-testnet.morphl2.io
-
-https://rpc2-testnet.morphl2.io
-~~~
+Morph Sepolia testnet is going to be sunset soon, we strongly suggest that you migrate to Morph Holesky testnet
```

Comment:
Morph Sepolia testnet is going to be sunset soon. We strongly suggest that you migrate to Morph Holesky testnet.
-- change "," to "."